app/models/manageiq/providers/inventory/persister.rb

```diff
@@ -106,7 +106,7 @@ def initialize_inventory_collections
   end
 
   def initialize_tag_mapper
-    @tag_mapper ||= ProviderTagMapping.mapper
+    @tag_mapper ||= ProviderTagMapping.mapper(:case_insensitive_labels => manager.try(:case_insensitive_labels?))
```

Comment:
As we discussed could we change this to:
1. The positive `:case_sensitive_labels`
2. Move this from `manager.try` to a method in the base persister that defaults to `true` and override in the azure persister to be `false`

Reply:
@agrare
I did suggested changes and there is [azure PR](https://github.com/ManageIQ/manageiq-providers-azure/pull/416)

thanks for the idea!